loadtest/store/memstore/random.go

```diff
@@ -65,7 +66,7 @@ func (s *MemStore) RandomTeamJoined() (model.Team, error) {
 
 	var teams []*model.Team
 	for teamId, team := range s.teams {
-		if s.teamMembers[teamId][userId] != nil {
+		if (s.teamMembers[teamId][userId] != nil) == memberOf {
```

Comment:
I would refactor the nil check to a separate variable called `isMember`, and then check `isMember == memberOf` to make this a bit more readable.

Reply:
Done, kind of :)